test/integration/connect/envoy/test-sds-server/Dockerfile

```diff
@@ -6,6 +6,6 @@ FROM golang:latest
 WORKDIR /go/src
 COPY ./ .
 
-RUN go build -v -o test-sds-server sds.go
+RUN go build -v -o test-sds-server ./...
```

Comment:
Required to vendor `grpr.go` (`GRPCLogger`) in `test-sds-server` to remove `consul` dependency and build all code.